trymito.io/pages/python-ai-tools.tsx

```diff
@@ -0,0 +1,223 @@
+                      Mito AI let me learn Python by doing my job. 
+                      I was able to automate my work using Mito, and it showed me how to do the same thing in Python. 
+                      I found that pattern matching prompts to generated code is a great way to learn.&quot;
+                    </i>
+                </p>
+                <p className={classNames('center')}> 
+                    — Kashvi, Bank Associate
+                </p>
+            </div>
+            <div className={pythonAIToolsStyles.case_study_headshot}>
+                <Image src='/KashviHeadshot.png' alt='Explore your data with Mito' width={250} height={250} ></Image>
+            </div>
+        </section>
+
+        <section>
+            <h1 className='center'>
+                Frequently Asked Questions
+            </h1>
+            <FAQCard title='How is Mito AI different from other AI copilots?'>
+                <ol>
+                    <li>
+                        AI chatbots are quickly becoming the fastest way to edit your data, but AI-generated code is often incorrect or has unwanted side effects. 
+                        For example, we&apos;ve seen users prompt LLMs to calculate a total_revenue column from a price * quantity columns. 
+                        The LLM may correctly give you the new column, but it may also delete the price and quantity columns from the dataset. 
+                        Since Mito users are often new to programming, they find it almost impossible to verify the code is correct just by reading it. 
+                        Instead, they need tooling that to help them check the correctness of their code that is built with their skillset in mind — spreadsheets!                   
+                    </li>
+                    <li>
+                        Since Mito is a spreadsheet, it knows the structure of your data, what column is selected, and your recent edits. 
+                        That context is required for Mito AI (we use ChatGPT under the hood) to generate useful code. But there&apos;s no reason that the user should be responsible for adding all of that additional information to their prompt. 
+                        Instead, Mito fills it all in for you. 
+                    </li>
+                    <li>
+                        Mito AI can be configured to use other LLMs. 
+                        This is especially important to security conscious enterprises that don&apos;t want to share any data with OpenAI. 
+                        Instead, they hook Mito up to their own On-Prem LLM.
+                    </li>
+                </ol>
+            </FAQCard>
+            <FAQCard title='What does Mito AI do with my data?'>
+                <div>
+                <p>
+                    Mito AI uses the instructions you provide (the prompt) and information about your dataframe to generate code that works in the context of your analysis. 
```

Comment:
<img width="1259" alt="Screen Shot 2023-07-05 at 2 28 06 PM" src="https://github.com/mito-ds/monorepo/assets/21002431/136b52c9-97d7-4889-bcc2-c2e05c2a8b05">

The spacing on both of these looks pretty strange. Want to space things a bit better? I think making the spacing smaller on the bullets in the second answer, and indenting the first bullets so the bullets are not in-line with the text?